api/charmdownloader/client_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package charmdownloader
+
+import (
+	gc "gopkg.in/check.v1"
+)
+
+type charmDownloaderSuite struct {
+}
+
+var _ = gc.Suite(&charmDownloaderSuite{})
+
+func (s *charmDownloaderSuite) TestWatchApplicationsWithPendingCharms(c *gc.C) {
+}
```

Comment:
Guessing this needs filling out?

Reply:
I should actually get rid of that file... It doesn't really do anything worth testing